package.json

```diff
@@ -44,12 +44,14 @@
       "dist/nodes/HttpBin/HttpBin.node.js"
     ]
   },
+  "dependencies": {
+    "n8n-workflow": "*"
```

Comment:
this is already listed in `peerDependencies`, and IMO that is the only place this should be listed in.